src/java.base/unix/classes/sun/nio/fs/UnixPath.java

```diff
@@ -25,11 +25,17 @@
 
 package sun.nio.fs;
 
-import java.nio.file.*;
-import java.nio.charset.*;
-import java.io.*;
+import java.nio.file.InvalidPathException;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.ProviderMismatchException;
+import java.nio.file.WatchEvent;
+import java.nio.file.WatchKey;
+import java.nio.file.WatchService;
+import java.nio.charset.CharacterCodingException;
+import java.io.IOException;
 import java.net.URI;
-import java.util.*;
+import java.util.Objects;
 
```

Comment:
These can be reverted; no changes needed in UnixPath.

Reply:
Reverted in ec0a341ef8186482c73c1909ffeaf061c26afc36.